sdk/iothub/Microsoft.Azure.Management.IotHub/tests/readme.md

```diff
@@ -0,0 +1,4 @@
+﻿# Running tests
+
+To run or record tests, you must set some environment variables.
```

Comment:
Since there are no recoded sessions for these tests, I assume this test doesn't run during the pipeline?
I would also mention here that these tests need to run and be validated upon any change, as they don't run during the pipeline.

Reply:
I'm working on recorded sessions. One of the tests is failing now and I'm tracking it down.

Yes, I'm running them by hand.